frontend/src/components/ClinicForm.js

```diff
@@ -0,0 +1,63 @@
+import { useState } from "react"
+import { useClinicContext } from '../hooks/useClinicContext'
+
+const ClinicForm = () => {
+    const {dispatch} = useClinicContext()
+    const [nazwa, setNazwa] = useState('')
+    const [numer_telefonu, setNumer_telefonu] = useState('')
+    const [adres, setAdres] = useState('')
+    const [error, setError] = useState(null)
+
+    const handleSubmit = async(c) => {
+        c.preventDefault()
+
+        const clinic = {nazwa, numer_telefonu, adres}
+
+        const response = await fetch('/api/clinics', {
+            method: 'POST',
+            body: JSON.stringify(clinic),
+            headers: {
+                'Content-Type': 'application/json'
+            }
+        })
+        const json = await response.json()
+
+        if(!response.ok){
+                setError(json.error)
+        }
+        if(response.ok){
+            setError(null)
+            setNazwa('')
+            setNumer_telefonu('')
+            setAdres('')
+            console.log('new clinic added', json)
```

Comment:
Please remove console.log statement. It is not needed to be put into the Production. We just use it for development purposes.